core/store/migrate/migrations/0128_add_multi_user.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+
+-- Create new user roles enum for users table
+CREATE TYPE user_roles AS ENUM ('admin', 'edit', 'edit_minimal', 'view');
```

Comment:
Do we expect these to change often?

Reply:
Could we brainstorm some alternative names for 'edit minimal'? :thinking: What exactly are we trying to capture here?
I see:
- Replay a specific block number
- Create job run

Do we expect to add more in the future?

Reply:
:shrug: 
```suggestion
CREATE TYPE user_roles AS ENUM ('admin', 'edit', 'run', 'view');
```
```suggestion
CREATE TYPE user_roles AS ENUM ('admin', 'edit', 'execute', 'view');
```
Also, wdyt about `read`/`write`?
```suggestion
CREATE TYPE user_roles AS ENUM ('admin', 'write', 'run', 'read');
```